easybuild/easyconfigs/b/Blitz++/Blitz++-0.10-foss-2016a.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'Blitz++'
+version = '0.10'
+
+homepage = 'http://blitz.sourceforge.net/'
+description = """Blitz++ is a (LGPLv3+) licensed meta-template library for array manipulation in C++
+ with a speed comparable to Fortran implementations, while preserving an object-oriented interface"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+sources = ['blitz-%(version)s.tar.gz']
+source_urls = ['https://sourceforge.net/projects/blitz/files/blitz/%(name)s %(version)s',
+               'download']
```

Comment:
I agree it looks wrong but the space is really there and is needed.  I've fixed the problems highlighted and updated the PR.  (I also added the easybuild file for `goolf-1.5.16` but maybe this should have been a separate PR?)